xwayland/xwm.c

```diff
@@ -940,6 +941,8 @@ static void xwm_handle_configure_notify(struct wlr_xwm *xwm,
 	xsurface->width = ev->width;
 	xsurface->height = ev->height;
 
+    wlr_signal_emit_safe(&xsurface->events.configure_notify, xsurface);
```

Comment:
One more nit: so far we've been cargo-culting the fact that we always provide a data pointer for events (here, with `xsurface`). This has the unfortunate consequence of always creating a breaking change each time we want to replace it with something useful.

Can we change it to `NULL`, so that we can easily change it to something useful later (e.g. a bitflag of properties that changed)?